pkg/pillar/cmd/baseosmgr/baseosmgr.go

```diff
@@ -64,14 +64,16 @@ type baseOsMgrContext struct {
 	pubZbootStatus           *pubsub.Publication
 
 	subGlobalConfig          *pubsub.Subscription
-	globalConfig             *types.GlobalConfig
+	//globalConfig             *types.GlobalConfig
 	subBaseOsConfig          *pubsub.Subscription
 	subZbootConfig           *pubsub.Subscription
 	subCertObjConfig         *pubsub.Subscription
 	subBaseOsDownloadStatus  *pubsub.Subscription
 	subCertObjDownloadStatus *pubsub.Subscription
 	subBaseOsVerifierStatus  *pubsub.Subscription
 	subNodeAgentStatus       *pubsub.Subscription
+	configItemValueMap		 *types.ConfigItemValueMap
+	configItemSpecMap		 *types.ConfigItemSpecMap
```

Comment:
Hmm.. Why do u need the specMap here?? Only Zedagent needs to use the SpecMap - isn't it? It uses it for parsing. Other agents only use the values published by zedagent..